terra_layer/management/commands/update_postgis_source.py

```diff
@@ -0,0 +1,176 @@
+    "commercantadressecommune": "commercantAdresseCommune",
+    "adresse_commercant": "adresse_commercant",
+    "commercantanneenaissance": "commercantanneenaissance",
+    "commercantstatut": "commercantStatut",
+    "commercantcessionfonds": "commercantCessionFonds",
+    "commercantcessionlocal": "commercantCessionLocal",
+    "commercantprojetentreprise": "commercantProjetentreprise",
+    "baillocatairebailtype": "bailLocataireBailType",
+    "baillocatairebailduree": "bailLocataireBailDuree",
+    "baillocatairebailfin": "bailLocataireBailFin",
+    "baillocataireloyer": "bailLocataireLoyer",
+    "baillocationvaleurlocative": "bailLocationValeurlocative",
+    "baillocataireloyercharges": "bailLocataireLoyerCharges",
+    "baillocataireloyertaxefonciere": "bailLocataireLoyerTaxeFonciere",
+    "baillocataireloyertaxefonciere_1": "bailLocataireLoyerTaxefonciere",
+    "baillocatairepasdeporte": "bailLocatairePasdeporte",
+    "baillocatairedroitaubail": "bailLocataireDroitaubail",
+    "baillocatairefondsdecommerce": "bailLocataireFondsdecommerce",
+    "vacancecommercesurfacevente": "vacanceCommerceSurfacevente",
+    "vacancecommercesurfacereserve": "vacanceCommerceSurfacereserve",
+    "vacancecommercelineaire": "vacanceCommerceLineaire",
+    "vacancecommerceactivite": "vacanceCommerceActivite",
+    "vacancecommercedate": "vacanceCommerceDate",
+    "vacancecommerceprixvente": "vacanceCommercePrixvente",
+    "vacancecommerceloyer": "vacanceCommerceLoyer",
+    "vacancecommercecharges": "vacanceCommerceCharges",
+    "vacancecommercedateproposition": "vacanceCommerceDateproposition",
+    "vacancecommercevisite": "vacanceCommerceVisite",
+    "vacancecommercecommentaire": "vacanceCommerceCommentaire",
+    "vacanceagencenom": "vacanceAgenceNom",
+    "vacanceagenceadressenum": "vacanceAgenceAdresseNum",
+    "vacanceagenceadresseindice": "vacanceAgenceAdresseIndice",
+    "vacanceagenceadressetypevoie": "vacanceAgenceAdresseTypevoie",
+    "vacanceagenceadresselibvoie": "vacanceAgenceAdresseLibvoie",
+    "vacanceadressecodepostal": "vacanceAdresseCodepostal",
+    "vacanceagenceadressecommune": "vacanceAgenceAdresseCommune",
+    "adresse_agence": "adresse_agence",
+    "vacancecontactidentite": "vacanceContactIdentite",
+    "vacancecontactidentitemail": "vacanceContactIdentiteMail",
+    "vacancecontactidentitetel": "vacanceContactIdentiteTel",
+    "vacancecontactadressenum": "vacanceContactAdresseNum",
+    "vacancecontactadresseindice": "vacanceContactAdresseIndice",
+    "vacancecontacttypevoie": "vacanceContactTypevoie",
+    "vacancecontactlibvoie": "vacanceContactLibvoie",
+    "vacancecontactcodepostal": "vacanceContactCodePostal",
+    "vacancecontactcommune": "vacanceContactCommune",
+    "adresse_vacancecontact": "adresse_vacancecontact",
+}
+
+
+class Command(BaseCommand):
+    help = "update a given layer with a given CSVSource"
+
+    def add_arguments(self, parser):
```

Comment:
You should add dry-run option to only show what's happens, just in case.

Reply:
Done.